app/models/ticket_purchase.rb

```diff
@@ -13,8 +13,12 @@ class TicketPurchase < ActiveRecord::Base
   delegate :price_cents, to: :ticket
   delegate :price_currency, to: :ticket
 
+  has_many :physical_tickets
+
   scope :paid, -> { where(paid: true) }
   scope :unpaid, -> { where(paid: false) }
+  scope :assigned, -> { where(assigned: true) }
```

Comment:
The assigbed scope is not currently used, right? So I would prefer introducing it when we need it, it as some point this happens